CMakeLists.txt

```diff
@@ -1,11 +1,11 @@
-cmake_minimum_required(VERSION 3.0.2)                        
-cmake_policy(SET CMP0048 NEW)
+cmake_minimum_required(VERSION 3.20)                        
+cmake_policy(SET CMP0074 NEW)
 
-project(xcompact3d LANGUAGES Fortran)
+project(Incompact3d LANGUAGES Fortran)
```

Comment:
Why not (X)xcompact3d?

Reply:
We can use Xcompact3d. I think we need to agree on a name and be coherent about it. No particular preference

Reply:
I think i kept Incompact3d because of the name of the repository since i tought that the name of the project is Incompact, but the exe is xcompact.

Reply:
xcompact3d please

Reply:
Ok I'll make sure there is the x everywhere.